era5_to_int.py

```diff
@@ -73,6 +73,83 @@ def consider(self, field, xlvl, proj, hdate, slab, intfile):
             self.t = None
             self.td = None
 
+class RHDiags:
```

Comment:
I'd suggest an extra blank line before this class definition to conform with the PEP 8 recommendations on blank lines.